src/main/java/org/jitsi/videobridge/cc/BandwidthProbing.java

```diff
@@ -32,16 +34,45 @@
     extends PeriodicRunnable
 {
     /**
+     * The system property name that holds a boolean that determines whether or
+     * not to activate the RTX bandwidth probing mechanism that implements
+     * stream protection.
+     */
+    public static final String
+        DISABLE_RTX_PROBING_PNAME = "org.jitsi.videobridge.DISABLE_RTX_PROBING";
```

Comment:
could we name this in the affirmative? `RTX_PROBING` or something?  the negative naming leads to weird double negatives like "if disable rtx probing is false..."